mailpoet/tests/docker/codeception/docker-entrypoint.sh

```diff
@@ -97,6 +97,9 @@ sed -i "s/define( *'WP_DEBUG', false *);/$CONFIG/" /wp-core/wp-config.php
 wp plugin activate woocommerce
 wp plugin activate woocommerce-subscriptions
 
+# activate theme
+wp theme activate twentytwentyone
```

Comment:
Can we/should we use 2022? (I think that's the default in WP 5.9.)

Reply:
We could and I tried. I decided not to do it because:

- We need to load a different theme when we want to test forms in widgets. 2022 does not have widget areas. So we would need more conditional actions depending on whether we are running acceptance-oldest and acceptance-latest.
- Some buttons and checkboxes on the front end become not interactable and the tests fail. The WC opt-in checkbox, the WC register button are a few examples. The tests pass manually, it is not an issue with our plugin. 

I can add a separate issue for it, I think it is not necessary to update the tests to 2022 now but we might need it when the work on the editor starts.

Reply:
Thanks for the explanation. I don't know what are the plans or how much we need it so I'll leave it up to you to decide whether this needs a ticket in the backlog.